txscript/sigcache_test.go

```diff
@@ -12,8 +12,58 @@ import (
 	"github.com/decred/dcrd/chaincfg/chainhash"
 	"github.com/decred/dcrd/dcrec/secp256k1/v3"
 	"github.com/decred/dcrd/dcrec/secp256k1/v3/ecdsa"
+	"github.com/decred/dcrd/wire"
 )
 
+// shortTxHashKey defines a short transaction hash key to be used throughout
+// the tests.
+var shortTxHashKey = func() [wire.ShortTxHashKeySize]byte {
+	shortTxHashKey, err := wire.CreateShortTxHashKey()
+	if err != nil {
+		panic(err)
+	}
+	return shortTxHashKey
+}()
+
+// msgTx113875_1 mocks the first transaction from block 113875.
+var msgTx113875_1 = func() *wire.MsgTx {
+	msgTx := wire.NewMsgTx()
+	txIn := wire.TxIn{
+		PreviousOutPoint: wire.OutPoint{
+			Hash:  chainhash.Hash{},
+			Index: 0xffffffff,
+			Tree:  wire.TxTreeRegular,
+		},
+		Sequence:        0xffffffff,
+		ValueIn:         5000000000,
+		BlockHeight:     0x3F3F3F3F,
```

Comment:
👍, copied this over from wire but should have updated. I can clean up the other cases like that at some point so that people don't copy from the places that have inconsistent formatting